app/src/components/login.rs

```diff
@@ -85,18 +85,16 @@ impl CommonComponent<LoginForm> for LoginForm {
                     server_data: res.server_data,
                     credential_finalization: login_finish.message,
                 };
-                self.common.call_backend(
-                    HostService::login_finish,
-                    req,
-                    Msg::AuthenticationFinishResponse,
-                )?;
+                self.common
+                    .call_backend(HostService::login_finish, req, move |r| {
+                        Msg::AuthenticationFinishResponse(r)
+                    })?;
                 Ok(false)
             }
-            Msg::AuthenticationFinishResponse(user_info) => {
+            Msg::AuthenticationFinishResponse(res) => {
+                let (r, b) = res.context("Could not log in")?;
```

Comment:
Can you use longer, more explicit names than r, b? I'm reviewing the PR and have already lost context on what they represent

Reply:
They should be something like response and success (?), but i'm not 100% sure what the bool is